README.md

```diff
@@ -9,4 +9,23 @@ Using TDD write Human vs Human game of Tic Tac Toe in java
 - 3x3 only
 - Console
 
-If you get stuck for more than an half-hour on how to write a test, ask for help.
+
+# Running the application
+
+1. Clone the repo {code} git@github.com:gemcfadyen/Apprenticeship-JavaTicTacToe.git {code}
```

Comment:
If you want to show code use the "back tick" `

i.e. \`git clone git@github.com:gemcfadyen/Apprenticeship-JavaTicTacToe.git\` would render as `git clone git@github.com:gemcfadyen/Apprenticeship-JavaTicTacToe.git`